db/mongo.js

```diff
@@ -0,0 +1,26 @@
+const { MongoClient } = require('mongodb');
+
+const dbName = 'abstgtplregistryva6';
+const url = process.env.MONGODB_URI || 'mongodb://localhost:27017';
+let db = null;
+
+async function connectToMongoDB() {
+  try {
+    const client = new MongoClient(url, { useNewUrlParser: true, useUnifiedTopology: true });
```

Comment:
Both `useNewUrlParser` and `useUnifiedTopology` are deprecated as of 4.0.0, and have no effect. We should remove them since we're using 6.5.0+